Sources/Clappr/Classes/Base/BaseObject.swift

```diff
@@ -129,11 +129,15 @@ open class BaseObject: NSObject, EventProtocol {
 }
 
 public extension BaseObject {
```

Comment:
Although this is an extension to `BaseObject`, I would move it to `Event.swift` as it is more about `Event` than about `BaseObject`. Thoughts?

Reply:
That’s a good point. But I don’t think this is a reason not to be close to BaseObject. For me, it feels like it should be here cause I’m extending this class behavior